### DISCUSSION TITLE
Mlfow 1.27.0 fails storing lstm model artefact with dropout set to 0.2

### DISCUSSION THREAD

**Discussion by kharepraveenLearning** [mlflow/mlflow#6838] [Q&A]
Hi,

Hello All,

I am using mflow 1.27.0  & facing issue in storing BI-LSTM model artefacts( into NFS)  with Dropout layer.  I am using tensorflow.keras libraries.

When I set Dropout as 0.2 it fails with below  error

dropout_per=0.2
model.add(Bidirectional(LSTM(lstm_neuron, dropout=dropout_per, recurrent_dropout=dropout_per,return_sequences=ret_seq)))

ValueError: Attempted to save a function b'__inference_forward_lstm_layer_call_fn_40681' which references a symbolic Tensor Tensor("dropout/mul_1:0", shape=(None, 100), dtype=float32) that is not a simple constant. This is not supported.

When I set Dropout as 0.0, it works 

Can anyone suggest a fix.

Regards
praveen

**Comment by JHibbard** [comment]
Hi Praveen,

Sounds like a frustrating issue, could you could provide a runnable example that I could help you debug? Thank you!

**Reply by kharepraveenLearning** [reply]
Hello JHibbard,

Thanks for reply. I will provide code for your reference. 

Regards
Praveen
